edx_repo_tools/drip_survey/upload.py

```diff
@@ -0,0 +1,59 @@
+"""
+
+"""
+import csv
+from datetime import date
+import hashlib
+import sys
+
+import click
+from edx_repo_tools.data import pass_repo_tools_data
+
+NAME = 'FirstName'
+EMAIL = 'PrimaryEmail'
+WEEK = 'Week'
+ASSOCIATED_WITH = 'AssociatedWith'
+UNSUBSCRIBED = 'Unsubscribed'
+
+def associated_with(person):
```

Comment:
@shaunagm: Does this make sense for dividing internal and external contributors (at least as far as the Eng Speed survey goes?)